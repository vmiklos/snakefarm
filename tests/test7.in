debug 1
out 1
time 100
load test7.map
addplayer 1
addsnake 1 1 0 1 1 0
25 0
24 0
endsnake
step
exit