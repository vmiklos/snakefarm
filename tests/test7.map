1;W;0;0
2;W;0;1
3;W;0;2
4;W;0;3
5;W;0;4
6;W;0;5
7;W;0;6

8;W;1;0
9;0;2;8
10;0;3;9
11;0;4;10
12;0;5;11
13;0;6;12
14;W;7;13

15;W;8;0
16;0;9;15
17;0;10;16
18;0;11;17
19;0;12;18
20;0;13;19
21;W;14;20

22;W;15;0
23;0;16;22
24;0;17;23
25;0;18;24
26;A;19;25
27;0;20;26
28;W;21;27

29;W;22;0
30;W;23;29
31;W;24;30
32;W;25;31
33;W;26;32
34;W;27;33
35;W;28;34

