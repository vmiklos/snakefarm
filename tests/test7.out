Event MapLoad test6.map
Event Step 1
StepEvent Snake_FieldBerry
End Step
Event Exit
